serialization/cbor.ksy

```diff
@@ -0,0 +1,328 @@
+meta:
+  id: cbor
+  title: Concise Binary Object Representation
+  endian: be
+  license: Unlicense
+  xref:
+    mime: application/cbor
+    rfc:
+      - 7049
+      - 8152
+      - 8610
+      - 8392
+    wikidata: Q28455556
+doc: |
+  A binary format capable of serialization of superset of JSON.
+seq:
+  - id: ib
+    type: ib
+  - id: payload
+    type:
+      switch-on: ib.major_type
+      cases:
+        'type::uint': uint
+        'type::neg_uint': uint
+        'type::byte_str': byte_str
+        'type::text_str': text_str
+        'type::array': array
+        'type::map': map
+        'type::tag': tag
+        'type::simple_or_float': simple_or_float
```

Comment:
`uint`, `array`, `map` would currently collide with similar-named constructs in many languages (see https://github.com/kaitai-io/kaitai_struct/issues/90). Can you consider adding certain prefix to your type names to make sure we escape the collision?